src/Digdir.Domain.Dialogporten.Infrastructure/HttpClientExtensions.cs

```diff
@@ -48,7 +48,8 @@ public static async Task<HttpResponseMessage> PostAsJsonEnsuredAsync(
             configureHeaders?.Invoke(httpRequestMessage.Headers);
             configureContentHeaders?.Invoke(httpRequestMessage.Content.Headers);
             var response = await client.SendAsync(httpRequestMessage, cancellationToken);
-            response.EnsureSuccessStatusCode();
+            await response.EnsureSuccessStatusCodeWithContent();
```

Comment:
Det er litt skummelt å logge response body for alle kontekster. Meg bekjent brukes denne extension for mer enn bare altinn auth, og det å logge respons (som kan være sensitiv) som en del av exception er litt så-som-så. Vi kan heller sette         "Serilog__MinimumLevel__Override__Digdir.Domain.Dialogporten.Infrastructure.Altinn.Authorization.AltinnAuthorizationClient": "Debug" i stage. Det har jeg gjort gjennom app configuration allerede.

Reply:
Nei, vi skal (ut i fra koden) få request body som vil si at vi kan reprodusere selv via postman.